bluebrain/repo-bluebrain/packages/py-libsonata/package.py

```diff
@@ -31,3 +31,4 @@ class PyLibsonata(PythonPackage):
     depends_on('py-numpy@1.17:', type=('build', 'run'))
     depends_on('py-setuptools', type='build', when='@0.1:')
     depends_on('py-setuptools-scm', type='build', when='@0.1:')
+    depends_on('mpi', when='^hdf5+mpi')
```

Comment:
```suggestion
    # Needed on Mac/clang
    depends_on('mpi', when='^hdf5+mpi platform=darwin')
```
We don't seem to need this on other platforms, which makes me wonder if there's something else going on. At the very least, this should be guarded.